ArcGISRuntimeSDKQt_CppSamples/Analysis/StatisticalQuery/StatisticalQuery.cpp

```diff
@@ -69,7 +69,7 @@ void StatisticalQuery::componentComplete()
   // Set map to map view
   m_mapView->setMap(m_map);
 
-  connect(m_featureTable, &ServiceFeatureTable::errorOccurred, [this](Error e)
+  connect(m_featureTable, &ServiceFeatureTable::errorOccurred, this, [this](Error e)
```

Comment:
I suppose that `m_featureTable` is deleted in the same time as `this` and then there is no risk that the connection is still alive with invalid `this` pointer. But it's easy to do a mistake.

And it's better to be consistent in the code and always have a third parameter.